src/edu/csus/ecs/pc2/ui/ProblemsPane.java

```diff
@@ -229,6 +231,10 @@ protected Object[] buildProblemRow(Problem problem) {
         // Object[] cols = { "Problem Name", "# Test Cases", "Input Method", "Judging Type", "Short", "Time Limit", "Input Validation", "I.V. Command", "Output Validator", "O.V. Command", "Groups" };
         // Object[] cols = { "Problem Name", "# Test Cases", "Input Method", "Judging Type", "Short", "Time Limit", "Input Validator", "I.V. Status", "Output Validator", "O.V. Command", "Groups" };
         // Object[] cols = { "Problem Name", "# Test Cases", "Input Method", "Judging Type", "Short", "Time Limit(secs)", "Max Output(KB)", "Input Validator", "I.V. Status", "Output Validator", "O.V. Command", "Groups" };
+        // Object[] cols = { "Problem Name", "# Test Cases", "Input Method", "Judging Type", "Short", "Time Limit", "Input Validator", "I.V. Status", "Output Validator", "O.V. Command", "Groups" };
+        // Object[] cols = { "Problem Name", "Test Cases", "Input Method", "Judging Type", "Short", "Time Limit", "Max Output(KB)", "Mem Limit(MB)",
+        //                      "Input Validator", "I.V. Status", "Output Validator", "O.V. Command", "Groups" };
```

Comment:
The purpose of this block of comments is to (1) preserve the history of how the table columns have changed, and (2) show the developer what the _current_ header looks like (so that there is a localized reference to show what method `buildProblemRow(Problem)` should be building.

However, something has gone wrong here; the above three added lines don't adhere to that convention.  First, the first of the three added lines is missing the `(secs)` indicator on the "Time Limit" field and is missing the `Max Output(KB)` field entirely.  Second, the last two lines (which are supposed to show the "current" value of the table column headers) are similarly missing the `(secs)` attribute on "Time Limit".  This could cause confusion for future developers.

Reply:
Evidently this was a result of a merge error.  The 1st line in the additions is the same as line 233; there was a merge done after the other two lines were added, and it preserved the original line (233).  I have removed that line and added the (secs) to the time limit column. Changed in [this commit.](https://github.com/pc2ccs/pc2v9/pull/732/commits/609d677d44262d1dbc6708fe0d923fc19db331d9)